src/main/java/com/github/fge/jsonpatch/MoveOperation.java

```diff
@@ -82,7 +85,7 @@ public JsonNode apply(final JsonNode node)
         if (movedNode.isMissingNode())
             throw new JsonPatchException(BUNDLE.getMessage(
                 "jsonPatch.noSuchPath"));
-        final JsonPatchOperation remove = new RemoveOperation(from);
+        final JsonPatchOperation remove = new RemoveOperation(from, NullNode.getInstance());
```

Comment:
Just for my understanding: What does this change do?

Reply:
It's needed because I added a parameter to the constructor of `RemoveOperation`. It's setting the `value` of the `RemoveOperation` to `null` because in this case we don't need a value.